Components/Hlms/Pbs/include/Cubemaps/OgreCubemapProbe.h

```diff
@@ -159,7 +170,8 @@ namespace Ogre
             This value allows you to override it with a different workspace definition.
         */
         void initWorkspace( float cameraNear = 0.5f, float cameraFar = 500.0f,
-                            uint8 mipmapsExecutionMask=0x01,
+                            TimeSlicing ts=TS_ONE_FRAME,
+                            uint32 executionMask=0xFFFFFFFF, uint32 mipmapsExecutionMask=0x01,
```

Comment:
This breaks the API in case someone doesn't use the default arguments; cannot do differently

Reply:
Even if `mipmapsExecutionMask` has gone because of IBL related work done by @darksylinc, `executionMask` has been kept as a nice side-effect (_it's feature_, cit.)